[
{
"cve_id": "CVE-2022-40897",
"patch_date": [
"2022-11-04 13:52:26-04:00"
],
"patch_files": [
"setuptools/package_index.py",
"setuptools/tests/test_packageindex.py"
],
"earliest_patch_date": "2022-11-04 13:52:26-04:00",
"earliest_vuln_date": "2017-02-24 19:37:39+02:00",
"vuln_files": [
"setuptools/package_index.py",
"setuptools/tests/test_packageindex.py"
],
"impact": {
"baseMetricV3": {
"cvssV3": {
"version": "3.1",
"vectorString": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:N/I:N/A:H",
"attackVector": "NETWORK",
"attackComplexity": "HIGH",
"privilegesRequired": "NONE",
"userInteraction": "NONE",
"scope": "UNCHANGED",
"confidentialityImpact": "NONE",
"integrityImpact": "NONE",
"availabilityImpact": "HIGH",
"baseScore": 5.9,
"baseSeverity": "MEDIUM"
},
"exploitabilityScore": 2.2,
"impactScore": 3.6
}
},
"cwes": [
"CWE-1333"
],
"fix_duplicate_type": false,
"fix_duplicate_types": [],
"intro_duplicate_type": false,
"intro_duplicated_types": [],
"file_fixed_before_fixed_again": false,
"files_fixed_before_fixed_again": [],
"file_fixed_before_vuln_again": false,
"files_fixed_before_vuln_again": []
},
{
"cve_id": "CVE-2024-6345",
"patch_date": [
"2024-04-29 20:01:38-04:00"
],
"patch_files": [
"setuptools/package_index.py",
"newsfragments/4332.feature.rst",
"setuptools/tests/test_packageindex.py",
"setup.cfg"
],
"earliest_patch_date": "2024-04-29 20:01:38-04:00",
"earliest_vuln_date": "None",
"vuln_files": [],
"impact": {},
"cwes": [],
"fix_duplicate_type": false,
"fix_duplicate_types": [],
"intro_duplicate_type": false,
"intro_duplicated_types": [],
"file_fixed_before_fixed_again": true,
"files_fixed_before_fixed_again": [
"setuptools/package_index.py",
"setuptools/tests/test_packageindex.py"
],
"file_fixed_before_vuln_again": false,
"files_fixed_before_vuln_again": []
}
]